packs/data/pathfinder-bestiary-3.db/rancorous-priesthood.json

```
@@ -793,12 +762,20 @@
                     "value": ""
                 },
                 "range": {
-                    "value": "60 feet"
+                    "value": ""
                 },
-                "rules": [],
+                "rules": [
+                    {
+                        "key": "ActiveEffectLike",
+                        "mode": "override",
+                        "path": "flags.pf2e.rollOptions.all.sonic-domain",
+                        "priority": 10,
+                        "value": true
+                    }
+                ],
```

```suggestion
                "rules": [],
```
No clue what that is for.